vendor/github.com/containerd/containerd/cio/io_unix.go

```diff
@@ -132,7 +132,7 @@ func openFifos(ctx context.Context, fifos *FIFOSet) (pipes, error) {
 			}
 		}()
 	}
-	if fifos.Stderr != "" {
+	if !fifos.Terminal && fifos.Stderr != "" {
```

Comment:
I see this fixes a potential panic; https://github.com/containerd/containerd/pull/3611, but I think that was related to a change that's only in 1.3.x, so I guess no need to get that into the 19.03 branch

Reply:
:+1:

Reply:
:+1: